lib/hanami/slice.rb

```diff
@@ -982,6 +982,11 @@ def load_router(inspector:)
             use(Hanami::Webconsole::Middleware, config)
           end
 
+          if Hanami.bundled?("hanami-router")
+            require "rack/method_override"
+            use(Rack::MethodOverride)
+          end
```

Comment:
```suggestion
          require "rack/method_override"
          use(Rack::MethodOverride)
```

I don't think we need the hanami-router bundled check here, because we're already inside a block being passed to `Slice::Router`, which itself has already required hanami/router (at the top of `lib/hanami/slice/router.rb`), so an extra check at this point is redundant.